ceph.spec.in

```diff
@@ -466,7 +488,17 @@ Requires:	librbd1 = %{epoch}:%{version}-%{release}
 Requires:	python-rados = %{epoch}:%{version}-%{release}
 Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
 %description -n python-rbd
-This package contains Python libraries for interacting with Cephs RADOS
+This package contains Python 2 libraries for interacting with Cephs RADOS
```

Comment:
Please propagate this change `s/Python libraries/Python 2 libraries/g` everywhere appropriate in both rpm and deb.